webpack.mix.js

```diff
@@ -24,4 +24,8 @@ mix.scripts([
 
 mix.scripts([
     './node_modules/eclipsefdn-solstice-assets/js/solstice.tables.js'
-], './static/js/solstice.hugo.js');
\ No newline at end of file
+], './static/js/solstice.hugo.js');
+
+mix.scripts([
+    './js/eclipsefdn.adopters.js',
+], './static/assets/js/eclipsefdn.adopters.js');
```

Comment:
Should we put this js in node_modules/  ? 🤔

Reply:
Yes :)

Reply:
https://github.com/EclipseFdn/solstice-assets/pull/200